wildfly-commonhaus-org/Team.md

```diff
@@ -0,0 +1,113 @@
+# WildFly Commonhaus Organization Team
+
+The people listed here currently act in various governance roles with the WildFly Commonhaus Organization.
+
+## Commonhaus Representative
+
+Brian Stansberry
+
+## Deputy Commonhaus Representative
+
+Thomas Jenkinson
+
+## Top Level Project Roles
+
+### Ironjacamar
+
+#### Project Lead
+
+Tomasz Adamski
+
+#### Maintainers
+
+TODO add link to Ironjacamar maintainers
+
+### JBeret
+
+#### Project Lead
+
+James Perkins
+
+#### Maintainers
+
+TODO add link to JBeret maintainers
+
+### JBoss Parent
+
+#### Project Lead
+
+David Lloyd
+
+#### Maintainers
+
+TODO add link to JBoss Parent maintainers
+
+### JBoss Threads
+
+#### Project Lead
+
+David Lloyd
+
+#### Maintainers
+
+TODO add link to JBoss Threads maintainers
+
+### JBoss WS
+
+#### Project Lead
+
+Richard Opalka
+
+#### Maintainers
+
+TODO add link to JBoss WS maintainers
+
+### Narayana
+
+#### Project Lead
+
+Thomas Jenkinson
+
+#### Maintainers
+
+TODO add link to Narayana maintainers
+
+### RESTEasy
+
+#### Project Lead
+
+James Perkins
+
+#### Maintainers
+
+TODO add link to RESTEasy maintainers
+
+### Undertow
+
+#### Project Lead
+
+Flavia Rainone
+
+#### Maintainers
+
+TODO add link to Undertow maintainers
+
+### Weld
+
+#### Project Lead
+
+Matěj Novotny
+
+#### Maintainers
+
+TODO add link to Weld maintainers
+
+### WildFly Application Server
+
+#### Project Lead
+
+Brian Stansberry
+
+#### Maintainers
+
+TODO add link to WildFly AS maintainers
```

Comment:
@bstansberry Do you want links to GitHub account, email address, or plain names like in the ones already there?

Reply:
Just an opinion, I think plain name and GitHub account given they'd have to have a GitHub account.

Reply:
The automation I wrote is described at https://github.com/bstansberry/git-file-adder?tab=readme-ov-file#gitorganizationwriters; script is in the same repo.

It writes Name, Github id, GitHub URL.

Thinking about it now, I'll tweak it so it combines the id and url into a markdown hyperlink. So name + github.

For WF AS at least, I think beyond the initial generation of a list of maintainers (which we can then edit) we just manually maintain it . The WF AS Governance doc says adding a maintainer is done via consensus process driven by a wildfly-dev discussion. So when that is done we update the maintainer list. If we decide to prune the list (e.g. if people are inactive) that also involves a discussion.